packages/camera/camera_platform_interface/lib/src/types/media_settings.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
It looks like this is missing the latest update from https://github.com/flutter/packages/pull/3586.

The [aggregate PR](https://github.com/flutter/packages/pull/3586/files#diff-66544a659722e66de98db82e1d79fbb08d6d217db22518f3235fbc1b8bbdda1aR1) has the documentation this is missing.

Reply:
Ok, i will merge